lib/PhpParser/PrettyPrinterAbstract.php

```diff
@@ -765,7 +765,13 @@ protected function p(
             $pos = $subEndPos + 1;
         }
 
-        $result .= $this->origTokens->getTokenCode($pos, $endPos + 1, $indentAdjustment);
+        $tokenCode = $this->origTokens->getTokenCode($pos, $endPos + 1, $indentAdjustment);
+        if ($node instanceof Expr\CallLike && $node->isFirstClassCallable()) {
+            $tokenCode = str_replace(',', '', $tokenCode);
```

Comment:
this can use `preg_replace()` with count arg = 1 to ensure only first `,` found that removed, see

https://3v4l.org/uZb6m

```suggestion
            $tokenCode = preg_replace('/,/', '', $tokenCode, 1);
```